bedrock/settings/base.py

```diff
@@ -1046,9 +1046,6 @@ def lazy_email_backend():
     lambda request: request.get_full_path() == '/healthz/'
 ]
 
-NEWRELIC_BROWSER_LICENSE_KEY = config('NEWRELIC_BROWSER_LICENSE_KEY', default='')
```

Comment:
Only thing I wasn't sure about here is if these settings are safe to remove. I can't see them being used anywhere outside of the JS snippet.